io/disk/scsi_remove_add.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Narasimhan V <sim@linux.vnet.ibm.com>
+
+# This Test removes and adds back a scsi device in all the specified PCI 
+# domains specified in the 'multiplexer' file.
+# Runs the test for '0000:01:00.0' if no value for pci domain is given.
+# This test needs to be run as root. 
+
+PATH=$(avocado "exec-path"):$PATH
+
+# Install dependencies
+if [[ `python -c 'from avocado.utils.software_manager import SoftwareManager; \
+    print SoftwareManager().install("lsscsi")'` == 'False' ]]
```

Comment:
You have to first test if it is installed. Otherwise a failure in internet access can exit this test as if the package were not present.